crates/walrus-service/src/client/cli/args.rs

```diff
@@ -854,6 +854,12 @@ pub struct AggregatorArgs {
     #[arg(long, num_args = 1.., default_values_t = default::allowed_headers())]
     #[serde(default = "default::allowed_headers")]
     pub(crate) allowed_headers: Vec<String>,
+    /// Whether to allow quilt patch tags to be returned in the response headers.
+    /// If true, the tags will be returned in the response headers, regardless of the allowed
```

Comment:
After reading the code, I think the `allowed_headers` are ignored if this config is true.